docs/organizations/using-a-coding-standard.md

```diff
@@ -92,6 +92,8 @@ To edit an existing coding standard or change the repositories that follow that
 !!! tip
     To ensure that all new repositories automatically follow the coding standard, [set the coding standard as default](#set-default).
 
+[//]: # (TODO ## Deleting a coding standard {: id="deleting"})
+
```

Comment:
I think this change was pushed by mistake @nicklem:

```suggestion
```

Reply:
You're right. I merged recently and this somehow snuck in. Reverting